src/routes/search.py

```diff
@@ -25,17 +25,25 @@ def get_blueprint():
     return search_blueprint
 
 
-@search_blueprint.route("/search_ingredients")
+
+@search_blueprint.route("/search_ingredients", methods=["POST"])
 def search_ingredients():
-    return render_template("search.html")
+    if flask.request.method == "POST":
+        ingredients = flask.request.form.get("searchIngredients")
+        # prints what the user searches for to the terminal
+        print(ingredients)
+        returnedDict = api.spoonacular.search_ingredients(ingredients)
+        # prints the 10 recipes to the terminal
+        print(returnedDict)
+        return render_template("index.html", ingredients=returnedDict)
+    return render_template("index.html")
 
 
 @search_blueprint.route("/search_recipes", methods=["POST"])
 def search_recipes():
     if flask.request.method == "POST":
```

Comment:
See comment above regarding importing flask. Update accordingly throughout the file.